### PR TITLE
Deprecate this exporter

### PR DESCRIPTION
**Pull request by roidelapluie**
https://groups.google.com/u/3/g/prometheus-developers/c/0yr5HI6va8M/m/Ftxw3mrcAgAJ

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]

**Comment by mrunge** [comment]
looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# Collectd Exporter [![Build Status](https://travis-ci.org/prometheus/collectd_exporter.svg)][travis]
+# !! This exporter is deprecated !!
 
-[![CircleCI](https://circleci.com/gh/prometheus/collectd_exporter/tree/master.svg?style=shield)][circleci]
-[![Docker Repository on Quay](https://quay.io/repository/prometheus/collectd-exporter/status)][quay]
-[![Docker Pulls](https://img.shields.io/docker/pulls/prom/collectd-exporter.svg?maxAge=604800)][hub]
+We recommend our users to use the [write_prometheus](https://collectd.org/wiki/index.php/Plugin:Write_Prometheus)
+plugin from collected instead of this exporter. This plugin is similar in scope
```
Comment (juliusv) [inline README.md:4]:
```suggestion
plugin from collectd instead of this exporter. This plugin is similar in scope
```

**Comment by juliusv** [comment]
Hm, did we decide on that now? My last PR on this still had some concerns from people around speed and TLS.

**Comment by juliusv** [comment]
The earlier PR was: https://github.com/prometheus/collectd_exporter/pull/96

**Comment by roidelapluie** [comment]
oh I missed that.

closing per #96

**Pull request closed without merging**